src/BridgedRoom.ts

```diff
@@ -322,6 +330,20 @@ export class BridgedRoom {
 
         // If we don't get an event then exit
         if (event === null) {
+            const reactionEntry = await this.main.datastore.getReactionByMatrixId(message.room_id, message.redacts);
+
+            if (reactionEntry) {
+                log.debug(`Trying to redact reaction "${reactionEntry.reaction}" on message ${reactionEntry.slackMessageTs} in channel ${reactionEntry.slackChannelId}`);
```

Comment:
I'd make this an info, it's probably low noise and would help people debug.

Reply:
I'll instead add an info.warn if the API call fails.